<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QLineEdit" name="leMask">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Маска файлов:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbDelete">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалять входные файлы</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePath">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPath">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>21</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbRewrite">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Заменять</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCounter">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавлять счетчик</string>
     </property>
    </widget>
    <widget class="QLabel" name="lDuplicate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>При повторении:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Путь для готовых:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbTimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Работа по таймеру</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="teTimer">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="pbAll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p% %v из %m</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p% %v мб из %m</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
